### PR TITLE
fixed 493, MkDocs encountered as error

### PR DESCRIPTION
**Pull request by longdafeng**
### Checklist

- [ ] I've run the tests to see all new and existing tests pass.
- [ ] If this Pull Request resolves an issue, I linked to the issue in the text above.
- [ ] I've informed the technical writer about the documentation change if necessary.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -1,29 +1,35 @@
...
 
 Before installing dependencies, please make sure you have installed a recent version of Python 3 and pip.
 
 Then you can run the following command in your terminal at current directory:
 
-    $ pip install -r requirements.txt
-
+```
+$ pip install -r requirements.txt
+$ pip install mkdocs-material
```
Comment (whhe) [inline docs/README.md:14]: I think it's better to add `mkdocs-material` into requirements.txt

Comment (longdafeng) [reply docs/README.md:14]: ok

**Review by whhe — APPROVED** [review]

**Review by LINxiansheng — APPROVED** [review]
LGTM

**Comment by LINxiansheng** [comment]
Thanks for your contribution

**Pull request merged**